.github/workflows/CI.yml

```
@@ -39,3 +39,14 @@ jobs:
           COMMIT_SHA: ${{ github.sha }}
         run: |
           while IFS= read -r line; do echo; sudo -E bash ./main.sh -vt "${line}" || exit 1; echo; done < <(git ls-files '.scripts/*.sh' | sed -E 's/^\.scripts\/(\w+)\.sh$/\1/')
+
+  mainfests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.2
+      - name: validate
+        env:
+          CI: true
+          COMMIT_SHA: ${{ github.sha }}
+        run: |
+          for image in $(grep -R "image:" compose | cut -d ":" -f3 | sed 's/^[[:space:]]*//g' | uniq | sort); do echo "Image: $image" && DOCKER_CLI_EXPERIMENTAL=enabled docker manifest inspect $image; done
```

```suggestion
          for image in $(grep -RPoh "image: \K.*" compose | sort -u); do echo "Image: $image" && DOCKER_CLI_EXPERIMENTAL=enabled docker manifest inspect $image; done
```
is a bit shorter. this is a fair idea but we do have deprecated app files still in the repo mostly for archival purposes. this would require removing those files which i have concerns about. if a user has not updated DS in a very long time and still has that app running when they upgrade DS tries to handle it gracefully and warn the user. removing the apps would result in no warnings and the app would just disappear. technically if a user already has an image pulled they could still run a container with it even after it is pulled from docker hub, however they would loose it if they stopped the container and purged images (functions that DS does have for users to be able to do). I'm open to additional discussion on this though.

---

I'm working around the deprecation issue now; it shouldn't matter when I'm done.